packages/web/src/pages/collection-page/CollectionPageProvider.tsx

```diff
@@ -143,11 +144,24 @@ type PlaylistTrack = { time: number; track: ID; uid?: UID }
 
 const CollectionPage = (props: CollectionPageProps) => {
   const currentTrack = useCurrentTrack()
-  return <CollectionPageClassComponen {...props} currentTrack={currentTrack} />
+  const tracks = useLineupTable(getCollectionTracksLineup)
+  return (
+    <CollectionPageClassComponen
+      {...props}
+      currentTrack={currentTrack}
+      tracks={tracks}
+    />
+  )
 }
 
 class CollectionPageClassComponen extends Component<
```

Comment:
nit: pre-existing but missing T lol
```suggestion
class CollectionPageClassComponent extends Component<
```